cli/provision.php

```diff
@@ -141,16 +148,24 @@ function abort(string $msg) {
     abort("$stackexistsmsg\n$help");
 }
 
+// check out latest moodle-fileconverter_librelambda-aws_stack
+$repo = "https://github.com/catalyst/moodle-fileconverter_librelambda-aws_stack.git";
+$stackdir = __DIR__ . '/../aws_stack';
+if (!is_dir($stackdir)) {
+    os_exec("git clone $repo $stackdir");
```

Comment:
This will likely fail for setups/containers where the application doesnt have write permission on its host directory. In this case, I think a tmpdir or request_dir will be better.